cmd/kube-scheduler/app/config/config.go

```diff
@@ -32,7 +32,7 @@ import (
 
 // Config has all the context to run a Scheduler
 type Config struct {
-	// config is the scheduler server's configuration object.
+	// ComponentConfig is the scheduler server's configuration object.
```

Comment:
Good catch, could you fix golint error in this package with this change?

Reply:
Thanks! At the beginning, I thought this was a small modification, so I submitted it together.
However, if I fix golint error for the whole package, I think it is beyond the scope of this PR :(
Would you mind if I send a PR separately to fix golint error?